js/console.ts

```diff
@@ -342,38 +342,125 @@ export function stringifyArgs(
   args: any[],
   options: ConsoleOptions = {}
 ): string {
-  const out: string[] = [];
-  const { collapsedAt, indentLevel } = options;
-  for (const a of args) {
-    if (typeof a === "string") {
-      out.push(a);
+  const first = args[0];
+  let a = 0;
+  let str = "";
+  let join = "";
+
+  if (typeof first === "string") {
+    let tempStr: string;
+    let lastPos = 0;
+
+    for (let i = 0; i < first.length - 1; i++) {
+      if (first.charCodeAt(i) === 37 /* '%' */) {
```

Comment:
I think we should move these constants (https://github.com/denoland/deno_std/blob/41a2d218264b2f9217bf793893aff5dadd2c4ca9/path/constants.ts#L41) from deno_std into the core repository and use constants instead of char codes

@ry what do you think?